src/forms/output/chartStringTransformers.js

```diff
@@ -2,32 +2,38 @@
  * @module */
 
 
-/** Recursively extract text from language-specific React
- *     objects. Creates one inline string of text.
- * Recursion is untested.
+/** Mutator. Set the thousands separator character for a Highcharts`
+ *     options.
  *
- * @params {object} translationObj React element containing
- *     children that are strings or other React elements.
+ * @params {object} translations The strings for the app's current
+ *     language.
+ * @params {object} Highcharts Where to set the options.
  *
- * @returns {string}
+ * @returns {nothing}
  */
-const textFromTranslatedElement = function (translationObj) {
-
-  const children = translationObj.props.children;
-  
-  if (typeof children === `string`) {
-    return children;
-
-  // To handle more complex translationObj objects
-  } else if (Array.isArray(children)) {
+const setThousandsSeparator = function (translations, Highcharts) {
+  // We really need our own number formatting functions
+  let separator = textFromTranslatedElement(translations.i_thousandsSeparator);
+  // This is for Highcharts to format pure numbers
+  Highcharts.setOptions({ lang: { thousandsSep: separator }});
```

Comment:
Since this is setting a universal Highcharts option, rather than a per-chart option, could this be called in a single "config global Highcharts" section, rather than on each graph? It looks like right now it's being called multiple times by each of the different Highcharts charts.

Reply:
I thought this was creating an instance of Highcharts, but if it is global, that's a great idea! @todo.